.sphinx/lxddoc/lxddoc.txt

```diff
@@ -0,0 +1,43 @@
+---
+orphan: true
+---
+
```

Comment:
To include the content in the docs, we'll need a marker before and after each group.
For example, `<!-- config group cluster start -->` and `<!-- config group cluster end -->`.